desc/geometry/surface.py

```diff
@@ -357,6 +359,67 @@ def from_near_axis(cls, aspect_ratio, elongation, mirror_ratio, axis_Z, NFP=1):
         surf = cls(R_lmn=R_lmn, Z_lmn=Z_lmn, modes_R=modes_R, modes_Z=modes_Z, NFP=NFP)
         return surf
 
+    @classmethod
+    def from_values(cls, coords, theta, M, N, NFP, sym=True, check_orientation=True):
```

Comment:
also just a reminder for @rahulgaur104 that you'll need to add the umbilic factor to this as well

Reply:
(not in this PR)